.github/workflows/ci.yml

```
@@ -13,4 +13,14 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
       - name: Run bash command
-        run: echo hello
\ No newline at end of file
+        run: echo hello
+build:
```

This needs to be nested inside of `jobs`
```suggestion
  build:
```

---

Doh. Thank you!

---

@prom-as Fixed the indentation, but I still have zero checks running.

---

OK I think I see the problem.

`error parsing called workflow
".github/workflows/ci.yml"
-> "PromiseNetwork/docker_tools_fork/.github/workflows/docker-build-image.yml@main"
: failed to fetch workflow: reference to workflow should be either a valid branch, tag, or commit`

Working on it.

---

@prom-as OK, got past the GHA issues and the checks are now running. Now I have a new perms error:

"Permission 'iam.serviceAccounts.getAccessToken' denied on resource (or it may not exist).

---

ooh yeah, one moment

---

@geoffplitt-promise , added this repo into the workload identity users, please retry